modules/core/src/main/scala/org/scalasteward/core/edit/UpdateHeuristic.scala

```diff
@@ -79,27 +79,34 @@ object UpdateHeuristic {
     getPrefixRegex = update => Some(s"${update.groupId}.*?")
   )
 
+  val sbt = UpdateHeuristic(
+    name = "sbt",
+    order = 2,
```

Comment:
I initially tried to set `order = 6` (largest), but could not.
Since the heuristic `original` has higher precedence than `sbt`.